command/update/command.go

```diff
@@ -196,56 +195,9 @@ func (c *Command) execute() error {
 			return microerror.Mask(err)
 		}
 
-		c.logger.Log("debug", fmt.Sprintf("added IP to endpoint of service '%s'", f.Kubernetes.Cluster.Service))
+		c.logger.Log("debug", fmt.Sprintf("added annotations to the KVM pod '%s'", f.Kubernetes.Pod.Name))
 	}
 
-	// Listen to OS signals issued by the Kubernetes scheduler.
-	signalCh := make(chan os.Signal, 2)
-	signal.Notify(signalCh, syscall.SIGTERM, syscall.SIGKILL)
-
-	c.logger.Log("debug", "waiting for termination signal")
-
-	s := <-signalCh
-
-	c.logger.Log("debug", fmt.Sprintf("received termination signal: %#v (%s)", s, s))
-
-	errCh := make(chan error)
-
-	// Use the updater to actually delete the endpoints identified by the provided
-	// flags.
-	go func() {
-		defer close(errCh)
-
-		action := func() error {
-			err := newUpdater.Delete(f.Kubernetes.Cluster.Namespace, f.Kubernetes.Cluster.Service, podInfos)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-			return nil
-		}
-
-		err := backoff.Retry(action, backoff.NewExponentialBackOff())
-		if err != nil {
-			errCh <- microerror.Mask(err)
-			return
-		}
-
-	}()
-
-	exitCode := 0
-
-	select {
-	case s := <-signalCh:
-		c.logger.Log("debug", fmt.Sprintf("received termination signal: %#v (%s) again, exiting immediately", s, s))
-	case err := <-errCh:
-		if err != nil {
-			c.logger.Log("error", fmt.Sprintf("%#v", microerror.Mask(err)))
-			exitCode = 1
-		}
-	}
-
-	c.logger.Log("debug", fmt.Sprintf("shutting down with exit code %d", exitCode))
-	os.Exit(exitCode)
-
-	return nil
+	// wait forever
+	select {}
```

Comment:
I always have problem recognising which image runs forever and which not. Since that changes I propose using a standard flag for that like `--wait-forever`. In that case it's obvious in using code that the image is supposed to run forever.

Reply:
Ok, that sounds reasonable, but I think we should propose such standard somewhere else than in some PR. Maybe somewhere on slack and then put it in news channel?

Reply:
+1 lets get it done here as it is and find a standard within a separate and more open scope.